.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+on: [pull_request]
```

Comment:
I will admit to not fully understanding the difference between the `push` and `pull_request` events.  Do you know which one (or both?) we should be using?

Reply:
I think [pull_request](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#pull-request-event-pull_request) is what we want. If I'm reading properly it will fire on updates to a PR, but not no pushes to branches that have no associated PR.

Reply:
Do we want these tests to run on master after we merge?  I would imagine we want to keep those, right?  Was chatting with @chingor13 and he made the great point we need that for codecov to have something to anchor off of when deciding delta.